resources/mod_php.rb

```diff
@@ -0,0 +1,36 @@
+include Apache2::Cookbook::Helpers
+unified_mode true
+
+property :module_name, String,
+         default: lazy { apache_mod_php_modulename },
+         description: 'Module name for the Apache PHP module.'
+
+property :so_filename, String,
+         default: lazy { apache_mod_php_filename },
+         description: 'Filename for the module executable.'
+
+property :package_name, String,
```

Comment:
We should allow this to be a string or array so that users can provide an array of packages if they need to